addons/stock/static/src/views/stock_orderpoint_list_controller.js

```diff
@@ -20,20 +25,32 @@ export class StockOrderpointListController extends ListController {
         if (action) {
             await this.actionService.doAction(action);
         }
+        if (model) {
+            return this.actionService.doAction({type: 'ir.actions.client', tag: 'reload'});
+        }
         return this.actionService.doAction('stock.action_replenishment', {
             stackPosition: 'replaceCurrentAction',
         });
```

Comment:
I.M.O. You don't need that the previous code with
```
return this.actionService.doAction('stock.action_replenishment', {
            stackPosition: 'replaceCurrentAction',
        });
 ``` add probably the same purpose to refresh the view.
 
You can replace everything by return this.actionService.doAction({type: 'ir.actions.client', tag: 'reload'}); only